beard/similarity/pairs.py

```diff
@@ -273,3 +274,40 @@ def transform(self, X):
         X2 = X[:, n_features:]
 
         return np.abs(X1 - X2)
+
+class SetJaccard(TransformerMixin):
+	"""Measure How much the 2 elements of a pair have in common"""
+    def __init__(self):
+        pass 
+    
+    def fit(self, X, y=None):
+	"""(Do nothing).
+
+        Parameters
+        ----------
+        :param X: array-like, shape (n_samples, n_features)
+            Input data.
+
+        Returns
+        -------
+        :returns: self
+        """
+        return self 
+    
+    def transform(self, X):    
+	"""get the intersection of the 2 co-authors set and compute the similarity as intersection over union
+
+        Parameters
+        ----------
+        :param X: array-like, shape (n_samples, n_features)
+            Input data.
+
+        Returns
+        -------
+        :returns: Xt which is an array of similarity values
+        """  
+
+        Xt= map(lambda x: float(len(x[0].intersection(x[1]))) / float(len(x[0].union(x[1]))) , X)
```

Comment:
Also, you function assumes that X is in fact `(n_samples, 2)` and that the two values are sets. I would rather assume that you have a `(n_samples, n_features)` array, where `X[i, :n_features/2]` is an array of number where all non-zero features encode for members of the set (e.g., like the representation yield by `CountVectorizer`).